AirWallex/Airwallex/Sources/AWXConstants.m

```diff
@@ -63,11 +63,11 @@
     }
 }
 
-Class ClassToHandleFlowForPaymentMethodType(NSString *type)
+Class ClassToHandleFlowForPaymentMethodType(AWXPaymentMethodType *type)
 {
-    if ([type isEqualToString:AWXCardKey]) {
+    if ([type.name isEqualToString:AWXCardKey]) {
         return NSClassFromString(@"AWXCardProvider");
-    } else if ([Airwallex.paymentFormRequiredTypes containsObject:type]) {
+    } else if (type.hasSchema) {
         return NSClassFromString(@"AWXPPROProvider");
```

Comment:
why `AWXPPROProvider`? it is not only for ppro

Reply:
> Do you have any idea to name it?

How about `CardPaymentHandler` and `DefaultPaymentHandler`?
if the type has schema, we can use DefaultPaymentHandler or we return error.
what is the behavior if the type does not have schema?

Reply:
PPRO is one of our provider. we also has Molpay, Wechat, Alipay, etc.,.
`AWXSchemaProvider` looks good to me.

Reply:
Cool!